developer/js/README.md

```diff
@@ -25,21 +28,28 @@ How to run the tests
 How to install
 --------------
 
-This will install both `kmlmc` and will let you install the compiler in
-other node projects on your machine.
+In a different repository, do this:
 
-    ./build.sh
-    npm link
+    npm install @keymanapp/developer-lexical-model-compiler
 
+If you're adding this to a subproject within
+<https://github.com/keymanapp/keyman>, use [lerna][] from the root
+directory:
+
+    # In root directory of the repository
+    npx lerna add @keymanapp/developer-lexical-model-compiler path/to/my-subproject
```

Comment:
Will this use the version in the local repository or fetch it from npmjs? Because internally I think we want to use the local working versions?

Reply:
This uses the version in the local repository; it does not fetch from npm package index.